go.mod

```diff
@@ -31,8 +31,8 @@ require (
 	google.golang.org/api v0.138.0
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/api v0.26.4
-	k8s.io/apimachinery v0.26.4
+	k8s.io/api v0.28.1
```

Comment:
We still cannot upgrade this:
https://github.com/operator-framework/operator-sdk/pull/6514
https://github.com/operator-framework/operator-sdk/issues/6381

https://github.com/dependabot close